<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">FlashKard</string>
  </property>
  <widget class="QWidget" name="mainButtonLayout">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QToolButton" name="loadButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>110</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Load a database of flashcards from disk.</string>
      </property>
      <property name="text">
       <string>L&amp;oad</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/document_open.svgz</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="testButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>110</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Open the tester and start learning the flashcards!</string>
      </property>
      <property name="text">
       <string>&amp;Test Me!</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/go_next.svgz</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="saveButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>110</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Save your current flashcards and progress to disk.</string>
      </property>
      <property name="text">
       <string>&amp;Save</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/document_save.svgz</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="databaseButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>110</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Open the editor: View, add, edit and remove flashcards.</string>
      </property>
      <property name="text">
       <string>&amp;Edit Database</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/edit.svg</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="statsButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>110</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Show a window with your score and other statistics.</string>
      </property>
      <property name="text">
       <string>&amp;View Stats</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/view_statistics.svgz</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="exitButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>110</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Close FlashKard.</string>
      </property>
      <property name="text">
       <string>E&amp;xit</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/application_exit.svgz</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFlashKard">
    <property name="title">
     <string>FlashKard</string>
    </property>
    <addaction name="actionOpen_FlashKard_Database"/>
    <addaction name="actionSave_FlashKard_Database"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Testing"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionStatistics"/>
    <addaction name="actionDatabase_Editor"/>
   </widget>
   <addaction name="menuFlashKard"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_FlashKard_Database">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/document_open.svgz</normaloff>:/images/document_open.svgz</iconset>
   </property>
   <property name="text">
    <string>&amp;Open FlashKard Database...</string>
   </property>
   <property name="statusTip">
    <string>Load a database of flashcards from disk.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_FlashKard_Database">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/document_save.svgz</normaloff>:/images/document_save.svgz</iconset>
   </property>
   <property name="text">
    <string>&amp;Save FlashKard Database...</string>
   </property>
   <property name="statusTip">
    <string>Save your current flashcards and progress to disk.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionStart_Testing">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/go_next.svgz</normaloff>:/images/go_next.svgz</iconset>
   </property>
   <property name="text">
    <string>Start Testing...</string>
   </property>
   <property name="statusTip">
    <string>Open the tester and start learning the flashcards!</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/application_exit.svgz</normaloff>:/images/application_exit.svgz</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string>Close FlashKard.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/view_statistics.svgz</normaloff>:/images/view_statistics.svgz</iconset>
   </property>
   <property name="text">
    <string>Statistics...</string>
   </property>
   <property name="statusTip">
    <string>Show a window with your score and other statistics.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDatabase_Editor">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/edit.svg</normaloff>:/images/edit.svg</iconset>
   </property>
   <property name="text">
    <string>Database Editor...</string>
   </property>
   <property name="statusTip">
    <string>Open the editor: View, add, edit and remove flashcards.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen_FlashKard_Database</sender>
   <signal>triggered()</signal>
   <receiver>loadButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDatabase_Editor</sender>
   <signal>triggered()</signal>
   <receiver>databaseButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>exitButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_FlashKard_Database</sender>
   <signal>triggered()</signal>
   <receiver>saveButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStart_Testing</sender>
   <signal>triggered()</signal>
   <receiver>testButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStatistics</sender>
   <signal>triggered()</signal>
   <receiver>statsButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
